doc/Development_Documentation/23_Installation_and_Upgrade/09_Upgrade_Notes/README.md

```diff
@@ -128,6 +129,12 @@ Please make sure to set your preferred storage location ***before*** migration.
 - [DataObjects\Documents] **BC Break**: Calling `getChildren/getSiblings` on `AbstractObject` or `Document` now returns unloaded listing. If the list is not traveresed immediately, then it is required to call `load()` explicitily.
   Also, `setChildren` now accepts `Listing` as first parameter instead of array.
 - [Admin] Removed `adminer` as built-in database management tool.
+- [Search] The search functionality has been extracted to its own bundle (`PimcoreSimpleBackendSearchBundle`)
+  - The `pimcore:search-backend-reindex` command has been moved to the search bundle
+  - Search icons all over Pimcore won't be available without the search bundle
+  - The "advanced" GDPR search has also been moved. We provide a basic search to cover the fundamental functionality if the search bundle isn't available.
+  - The asset, object, document and quick search have been moved to the search bundle
+  - All backend-search related files have been moved to the search bundle (please check custom implementations if you rely on any backend-search component!)
```

Comment:
missing: 
- additional messenger transport for backend search

Reply:
done 9d4e90685f56b93ff815b9681ae4ac8dac637c28